### ISSUE TITLE
redundant information filter

### ISSUE THREAD

**Issue by Karrenbelt** [matthiaskoenig/brendapy#30]
there is some redundant information stored in BRENDA (duplicate km, ki, kcats). It would be nice to have the option to filter these, for which I suggest:

def remove_duplicate_dicts_from_list(l : List[Dict]):
    new_l = []
    for i in range(len(l)):
        if l[i] not in l[i + 1:]:
            new_l.append(l[i])
    return new_l

let me know what you think, or if I'm wrong about the redundant information

**Comment by matthiaskoenig** [comment]
@Karrenbelt Thanks for reporting this. Could you provide some concrete example, i.e., ec and protein id? This will make it much easier to implement and test this.

**Comment by Karrenbelt** [comment]
certainly, the first one I run into if I parse all E.C. numbers is E.C. 1.1.1.1 for Mus Musculus KI's. It contains 27 entries, of which only 9 are unique. I just tested how often this occurs: 5674 times, and interestingly all of them are multiples of 3.

I had to zip the pickled object to be able to attach it
[example.p.zip](https://github.com/matthiaskoenig/brendapy/files/3604077/example.p.zip)
